### DISCUSSION TITLE
Error: "The schema must be one of the following: public" is showing when doing the Quickstart guide of ToolJet

### DISCUSSION THREAD

**Discussion by peterpardo** [ToolJet/ToolJet#11422] [Q&A]
I was trying to fork the Tooljet repo to run it in my local machine and try contribute and fix some issues. This is my first attempt trying to contribute to open source projects.

I ran the docker script in the README.md** and was able to make it work in my local machine. Then, I tried to follow the [Quickstart Guide](https://docs.tooljet.com/docs/getting-started/quickstart-guide) to familiarize myself with the tool.

While doing the 2nd step of the guide, which is [Create a Database Table](https://docs.tooljet.com/docs/getting-started/quickstart-guide#2-create-a-database-table), it showed me this error when I tried to create the table:

![image](https://github.com/user-attachments/assets/0ece4f78-b7c8-406c-85d4-d5075ef7e9f2)

![image](https://github.com/user-attachments/assets/0bc3413a-4817-4155-a654-a5655b03b46e)

Is there anything that I missed while setting up the app in my local machine?

Thank in advance for the help and have a wonderful day!

**Comment by han8435762** [comment]
兄弟 你解决了吗

**Comment by PriteshKiri** [comment]
Hey @peterpardo,

Can you please drop your query in the general channel of our [Slack](https://www.tooljet.com/slack) community?

Our team would be happy to assist you there.

**Comment by timthedevguy** [comment]
Hey @peterpardo did they get you resolved?  I'm having the same issue but a custom docker compose setup

**Reply by PriteshKiri** [reply]
Hey @peterpardo,

Can you please drop your query in detail in the general channel of our [Slack](https://www.tooljet.com/slack) community?

**Reply by qimiao032** [reply]
same issue
